Dockerfile

```diff
@@ -1,15 +1,25 @@
-FROM python:3.9
+FROM quay.io/centos/centos:stream8
 
+RUN dnf -y module install python39 && dnf -y install python39 python39-pip git
 RUN mkdir /app
+ADD LICENSE /app
 ADD wait_plugin.py /app
 ADD test_wait_plugin.py /app
 ADD requirements.txt /app
 WORKDIR /app
 
-RUN pip install -r requirements.txt
-RUN /usr/local/bin/python3 test_wait_plugin.py
+RUN pip3 install -r requirements.txt
+RUN python3 -m coverage run test_wait_plugin.py
+RUN python3 -m coverage html -d /htmlcov --omit=/usr/local/*
 
 VOLUME /config
 
-ENTRYPOINT ["/usr/local/bin/python3", "/app/wait_plugin.py"]
-CMD ["-f", "/config/wait.yaml"]
\ No newline at end of file
+ENTRYPOINT ["python3", "/app/wait_plugin.py"]
+CMD []
+
+LABEL org.opencontainers.image.source="https://github.com/arcalot/arcaflow-plugin-wait"
```

Comment:
Still says `wait`.

I have lots of reservations about separate repositories still.